template/package.json

```diff
@@ -16,7 +16,7 @@
     "reconnect:android": "adb reverse tcp:8081 tcp:8081 && adb shell input text 'RR'",
     "test": "jest",
     "typescript": "tsc -p ./tsconfig.json --pretty --noEmit --skipLibCheck",
-    "lint": "eslint . --ext .js,.jsx,.ts,.tsx"
+    "lint": "eslint . --ext .js,.jsx,.ts,.tsx --fix"
```

Comment:
I'm not sure whether we should use the `--fix` flag here. I assume everyone runs `eslint` automatically in their IDE and this script is just for checking on the CI server. There we shouldn't be fixing things (but just checking), because they're not committed afterwards.